webapi/src/test/scala/org/knora/webapi/store/triplestore/upgrade/plugins/ApplyUpgradePluginToTestData.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.store.triplestore.upgrade.plugins
+
+import org.apache.jena.riot.RDFDataMgr
+import org.apache.jena.riot.RDFFormat
+import zio.Scope
+import zio.ZIO
+import zio.ZIOAppDefault
+
+import java.io.FileInputStream
+import java.io.FileOutputStream
+import java.io.InputStream
+import java.nio.file.Files
+import java.nio.file.Path
+import scala.jdk.CollectionConverters.IteratorHasAsScala
+
+import org.knora.webapi.messages.util.rdf.RdfFeatureFactory
+import org.knora.webapi.messages.util.rdf.RdfModel
+import org.knora.webapi.messages.util.rdf.Turtle
+import org.knora.webapi.messages.util.rdf.jenaimpl.JenaConversions._
+import org.knora.webapi.store.triplestore.upgrade.UpgradePlugin
+
+object ApplyUpgradePluginToTestData extends ZIOAppDefault {
```

Comment:
Correct, not in my branch. Since this code is already there and works I thought about keeping it around? Might as well remove it but then it is lost forever.

Reply:
The `ApplyUpgradePluginToTestData` worked in the sense that it updated the test data correctly. I have just decided not to change the base ontology because this change is too hard to do as the existing tests broke in unforseen ways and fixing the code was too hard. The change to the base ontology is not necessary for this route to work.